lib/getLocalIdent.js

```diff
@@ -6,8 +6,15 @@ var loaderUtils = require("loader-utils");
 var path = require("path");
 
 module.exports = function getLocalIdent(loaderContext, localIdentName, localName, options) {
-	if(!options.context)
-		options.context = loaderContext.options && typeof loaderContext.options.context === "string" ? loaderContext.options.context : loaderContext.context;
+	if(!options.context) {
```

Comment:
Can we solve this in one line?

Reply:
I could refactor it to nested ternary expressions:
`options.context = options.context || (loaderContext.rootContext ? loaderContext.rootContext : loaderContext.options && typeof loaderContext.options.context === "string" ? loaderContext.options.context : loaderContext.context)`

Reply:
@yavorsky it is better in future to maintenance, just add or remove new options/context and etc

Reply:
But we could just remove condition case.
Nested ternary expressions are not allowed by current eslint config.

Reply:
let's wait @michael-ciniawsky 

Good work! Thanks!